sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CosmosDB Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CosmosDB Resources.")]
 
-[assembly: AssemblyVersion("3.5.0.0")]
-[assembly: AssemblyFileVersion("3.7.0.0")]
+[assembly: AssemblyVersion("3.9.0.0")]
+[assembly: AssemblyFileVersion("3.9.0.0")]
```

Comment:
since your SDK is 
For minor upgrade, you only need to bump `AssemblyFileVersion` and you **don't** need to bump the `AsemblyVersion` if it's just a minor upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> 6. Update the following information in AssemblyInfo.cs
      - AssemblyFileVersion (Must have)
      - AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Reply:
rolled back the AsemblyVersion upgrade